kroxylicious-api/src/main/java/io/kroxylicious/proxy/metadata/DescribeTopicLabelsRequest.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.metadata;
+
+import java.util.Collection;
+
+/**
+ * A request to describe the labels of the given topics.
+ * @param topicNames
+ */
+public record DescribeTopicLabelsRequest(Collection<String> topicNames) implements ResourceMetadataRequest<DescribeTopicLabelsResponse> {}
```

Comment:
I'm wondering if we should have an intermediate object ( DescribeTopicLabelsRequestData) representing the request to give us how we make the request.

Reply:
Could you elaborate on this point @k-wall, I'm not sure I understand the point you're making/benefit of having an extra object.